src/AqueductV1Pair.sol

```diff
@@ -128,7 +140,7 @@ contract AqueductV1Pair is IAqueductV1Pair, AqueductV1ERC20, SuperAppBase {
      * @return reserve0 The reserve of token0 at the specified time.
      * @return reserve1 The reserve of token1 at the specified time.
      */
-    function getReservesAtTime(uint32 time) public view returns (uint112 reserve0, uint112 reserve1) {
+    function getReservesAtTime(uint32 time) public view override returns (uint112 reserve0, uint112 reserve1) {
         uint112 totalFlow0 = uint112(uint96(cfa.getNetFlow(token0, address(this))));
         uint112 totalFlow1 = uint112(uint96(cfa.getNetFlow(token1, address(this))));
```

Comment:
Should these be `uint96` now? Same with params for internal `_getReservesAtTime` function

Reply:
@joshkokatnur what did you think of this comment? Also the two subsequent comments

Reply:
agreed, added this in 970649a